usaspending_api/etl/es_recipient_profile_template.json

```diff
@@ -0,0 +1,71 @@
+{
+  "settings": {
+    "index.mapping.ignore_malformed": true,
+    "index.max_result_window": null,
+    "index.refresh_interval": -1,
+    "index": {
+      "number_of_shards": 5,
+      "number_of_replicas": 0,
+      "sort.field": [
+        "recipient_name.keyword",
+        "uei.keyword"
+      ],
+      "sort.order": [
```

Comment:
Is the sort to support sorting and paging results for the API queries

Reply:
`index.sort.field` defines which field(s) to sort by when indexing documents and `index.sort.order` defines what order (asc or desc) to sort the field(s) by. These settings just affect how Elasticsearch internally organizes the terms and has nothing to do with how results are returned when a query is executed. They're just related to how ES divides up the records across the shards in the cluster.

As part of a query, Elasticsearch returns documents based on a relevance score named `_score` and the results are returned in descending order based on their `_score` value.

Reply:
Removed these sort options since they aren't going to help the performance during index time and we aren't sorting our results in the same way, so there's no benefit to having these values.